core/src/avm2/globals/flash/display3D/Context3DProfile.as

```diff
@@ -15,15 +16,19 @@ package flash.display3D
         public static const BASELINE_CONSTRAINED:String = "baselineConstrained";
 
         // Use an extended feature support profile to target newer GPUs which support larger textures This profile increases the maximum 2D Texture and RectangleTexture size to 4096x4096
+        [API("690")]
         public static const BASELINE_EXTENDED:String = "baselineExtended";
 
         // Use an standard profile to target GPUs which support MRT, AGAL2 and float textures.
+        [API("698")]
         public static const STANDARD:String = "standard";
 
         // Use an standard profile to target GPUs which support AGAL2 and float textures.
+        [API("702")]
         public static const STANDARD_CONSTRAINED:String = "standardConstrained";
 
         // Use standard extended profile to target GPUs which support AGAL3 and instanced drawing feature.
+        [API("704")]
         public static const STANDARD_EXTENDED:String = "standardExtended";
```

Comment:
I think this should be `706` for Flash Player 18:
https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/display3D/Context3DProfile.html#STANDARD_EXTENDED